app/models/book.rb

```diff
@@ -0,0 +1,7 @@
+class Book < ApplicationRecord
+  belongs_to :user
+  has_many :posts
+  has_many :comments
+  has_many :book_categories
```

Comment:
làm cách nào em check những hàm này được viết đúng hay không?.

Reply:
thì lúc mình seed dữ liệu, ví dụ ở bảng book này, nếu như book.user mà lấy được toàn bộ thông tin user thì tức là nó đã nối được vào với nhau ạ